Jenkinsfile

```diff
@@ -141,9 +141,10 @@ def daos_packages_version(String distro) {
 }
 
 def fault_test_tag() {
-    if !(commitPragma(pragma: 'Skip-fault-test').contains('true'))
-        return ",faults"
-    return ",-faults"
+    if (cachedCommitPragma(pragma: 'Skip-fault-test').contains('true')) {
```

Comment:
Thanks!  I included that change...

Reply:
As a point of process, you should reserve resolving conversations for the person who started the conversation, so that they can confirm the result of a conversation is as they intended.